UndertaleModTool/Scripts/Community Scripts/TouchControlsEnabler.csx

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+string display_name = Data?.GeneralInfo?.DisplayName?.Content.ToLower();
+
+if (display_name != "deltarune chapter 1 & 2" && display_name != "deltarune chapter 1&2" && display_name != "undertale")
+{
+    ScriptError("Error 0: Only compatible with Undertale & Deltarune");
+    return;
+}
+
+string dataPath = Path.Combine(Path.GetDirectoryName(ScriptPath), "TouchControls_data");
+
+int lastTextPage = Data.EmbeddedTextures.Count - 1;
+int lastTextPageItem = Data.TexturePageItems.Count - 1;
```

Comment:
these two seem unused now and could be removed.

Reply:
aight.